cypress/features.md

```diff
@@ -16,7 +16,7 @@ The following features will be implemented. This gets done in three steps:
 
 ### User Account
 
-[Cucumber Features](./integration/user_account)
+[Cucumber Features](https://github.com/Human-Connection/Human-Connection/tree/master/cypress/integration/user_account)
```

Comment:
```suggestion
[Cucumber Features](./integration/user_account)
```
In this case it would be better to keep the relative path. Why? If you send in a PR and let's say you changed the referenced file and if a code reviewer would click on the link, she would be redirected to the `master` branch. You can try it out yourself by clicking on "Preview" and then click on the link.

Reply:
You're right, that's not ideal. The reason I changed it is that the relative paths are not working on GitBook. For some reason they add a `/README.md` to the end of the link and since there are no `README`s in the `cypress/integrations` folder you land on a 404 page...

Any idea how to get this working for both use cases?

Reply:
No! In that case I'll approve this and merge. :+1: